book/intro.md

```diff
@@ -1,11 +1,121 @@
-# Welcome to your Jupyter Book
+# 🥳 Welcome to the EGU22 PyGMT short course
```

Comment:
```suggestion
# Crafting beautiful maps with PyGMT

## 🥳 Welcome to the EGU22 PyGMT short course
```

The h1 in this page is picked up as the title for the Jupyter book, which maybe should be the title of the course? Then we can have the welcome message as an h2 beneath it.

Reply:
Sounds good, done in f0592acf249d3543f41222b84cb93c5e646ae21a